src/scripts/rabbitmq.coffee

```diff
@@ -21,10 +21,9 @@
 # Notes:
 #
 # Author:
-#   kevwil
+#   kevwil, davidsulpy
 
-
-host = process.env.HUBOT_RABBITMQ_HOST
```

Comment:
Looks like a mismatch here between what's documented to use (HUBOT_RABBITMQ_ROOT_URL) and what is used from the code (HUBOT_RABBITMQ_HOST)